src/candy-machine/candy-machine.service.ts

```diff
@@ -421,6 +421,7 @@ export class CandyMachineService {
             isSigned: item.isSigned,
             rarity: PrismaComicRarity[ComicRarity[item.rarity].toString()],
             collectionName: onChainName,
+            collectionAddress: collectionNftAddress.toString(),
```

Comment:
should we rename `collectionNftAddress` that's coming from `getOrCreateComicIssueCollection` to `collectionAddress`?